build-system/server/lazy-build.js

```diff
@@ -17,6 +22,9 @@ const {VERSION} = require('../compile/internal-version');
 const extensionBundles = {};
 maybeInitializeExtensions(extensionBundles, /* includeLatest */ true);
 
+const componentBundles = {};
```

Comment:
nit: unclear the distinction between "extension" and "component". The pattern you followed elsewhere has also been to use the word "bento" as a replacement for "extension"

```suggestion
const bentoBundles = {};
```

Reply:
That does seem like a better name, however, I have been using the word component to refer to bento components so while this really isn't inconsistent, I will go ahead and clarify in the other situations.